### PR TITLE
Updates for running with 0.3 API GW

### PR DESCRIPTION
**Pull request by oddjobz**
This introduced dependencies based on the new common code but in particular on the API GW. This version needs to run against an 0.3+ version of the GW. I've also removed the old security code that we're no longer using (from the swagger file) which makes the startup far less noisy ..)

### CODE REVIEW

File: config.ini [hunk 1]
```diff
@@ -92,6 +96,10 @@ db_connection = ${db_driver}://${db_user}:${db_pass}@${db_host}:${db_port}/${db_
 db_schema = ras
 
 [demo]
+pi_protocol = https
```
Comment (jonmorphic) [inline config.ini]: Typo here?

**Review by jonmorphic — CHANGES_REQUESTED** [review]
Looks straightforward, just a possible typo in demo section.

**Comment by jonmorphic** [comment]
A further comment on this, but one to deal with out-of-band of this PR. How can I tell which is the latest stable version of ras-common? This branch pulls from a feature branch version, is that what you intended?

**Comment by oddjobz** [comment]
Ok, fixed typo, and have changed the requirements.txt .. branching obviously subject to some dicussion but for now it's tagged to the 'release' branch.

**Review by jonmorphic — APPROVED** [review]

**Comment by jonmorphic** [comment]
I did approve these changes on the basis the tests pass for me. However, I'm unsure where the syntax has come from for the ons-ras-common requirement, as it looks incorrect on closer inspection - it's actually going to pull from the head of the release/0.3 branch. I've gone down the requirements-from-git path before and it ended up a mess. I suspect tagging the version is the best we can do right now until a PyPI host can be setup.

Edit: meant to point out the Travis test failures. That doesn't happen for me locally.

**Pull request closed without merging**